recipes-tag/xpdacq/meta.yaml

```diff
@@ -44,7 +45,7 @@ test:
 about:
   home: https://github.com/xpdAcq/xpdAcq
   license: BSD-3-Clause
-  license_family: BSD
+  license_family: BSD 
   license_file: LICENSE
```

Comment:
blanks space here and below?

Reply:
This is from the conda-forge recipe. At this point I'd accept that and move on. At some point I'll fix it in the conda-forge package and it will be fixed here.

Reply:
Thanks, I see.